torch/_C/__init__.pyi.in

```diff
@@ -293,8 +293,14 @@ class Node:
 
 
 # Defined in torch/aten/src/ATen/core/function_schema.h
+class Argument:
+    name: str
+    type: str
+    default: str
+    def has_default_value(self) -> _bool: ...
 class FunctionSchema:
-    ...
+    arguments: List[Argument]
+    returns: List[Argument]
```

Comment:
nit: keep ellipsis to indicate that the definition is not complete?
```suggestion
    arguments: List[Argument]
    returns: List[Argument]
    ...
```

Reply:
Right, I've added ellipsis.